target/linux/pistachio/config-4.4

```diff
@@ -520,14 +519,7 @@ CONFIG_USB_DWC2_DUAL_ROLE=y
 CONFIG_USB_EHCI_HCD=y
 # CONFIG_USB_EHCI_HCD_PLATFORM is not set
 # CONFIG_USB_ETH is not set
-CONFIG_USB_EZUSB_FX2=y
-CONFIG_USB_F_MASS_STORAGE=y
 CONFIG_USB_GADGET=y
-CONFIG_USB_HID=y
-CONFIG_USB_LIBCOMPOSITE=y
-CONFIG_USB_MASS_STORAGE=y
-CONFIG_USB_MON=y
-CONFIG_USB_OTG=y
```

Comment:
Note this was removed as we can still boot without it. It seems the dwc2 doesn't actually use this config value however some parts of the usb core are switched off but again nothing that seemed to affect our current use cases.